patches/third_party-tflite_support-BUILD.gn.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/third_party/tflite_support/BUILD.gn b/third_party/tflite_support/BUILD.gn
+index 214e39238ae4a52aa42775916062423e872e8b82..29b5180f362458ec25308ef2831b011dd825c347 100644
+--- a/third_party/tflite_support/BUILD.gn
++++ b/third_party/tflite_support/BUILD.gn
+@@ -61,6 +61,7 @@ flatbuffer("metadata_schema") {
+   # now causes a warning message with flatc.
+   args = [ "--no-warnings" ]
+ }
++import("//brave/chromium_src/third_party/tflite_support/sources.gni") flatbuffer_text_embedder_fix("encoder_config") {}
+ 
+ static_library("tflite_support") {
+   sources = [
+@@ -162,6 +163,7 @@ static_library("tflite_support") {
+     "//third_party/abseil-cpp:absl",
+     "//third_party/re2",
+   ]
++  import("//brave/chromium_src/third_party/tflite_support/sources.gni") sources += text_embedder_fix_sources deps += text_embedder_fix_deps
```

Comment:
see https://github.com/brave/brave-core/pull/24953/files#r1716128084

Reply:
`tflite_support` doesn't depend on the sources we are adding here so it seems like a target in brave that includes them would eliminate the need for this?

Reply:
https://github.com/brave/brave-core/pull/24953#discussion_r1717376626